gc3libs/__init__.py

```diff
@@ -1816,7 +1816,7 @@ def fget(self):
             return self.history.last()
 
         def fset(self, value):
-            self.history.append(unicode(value))
+            self.history.append(unicode(value, errors='replace'))
```

Comment:
Ok, this avoids raising exceptions here -- but why are there invalid characters in `value` in the first place?

The only reason I can think of is that a file name or other user-inputted string was interpolated into `value` with the wrong encoding.  So we're silently suppressing errors instead of fixing the root problem; not something I'm really fond of...